code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -418,6 +418,14 @@
 	G.key = ghost.key
 	G << "You are an adamantine golem. You move slowly, but are highly resistant to heat and cold as well as blunt trauma. You are unable to wear clothes, but can still use most tools. Serve [user], and assist them in completing their goals at any cost."
 	qdel(src)
+	var/datum/objective/protect/new_objective = new /datum/objective/protect
+	new_objective.owner = G:mind
+	new_objective:target = usr:mind
```

Comment:
```:```'s are a big no no now we have standards.
Primarily because ```:``` can ALWAYS be avoided, with type casting.